src/PostgreSql/docs/readme.md

```diff
@@ -8,4 +8,4 @@ This directory contains the documentation of the cmdlets for the `Az.PostgreSql`
 - Packaged: yes
 
 ## Details
-The process of documentation generation loads `Az.PostgreSql` and analyzes the exported cmdlets from the module. It recognizes the [help comments](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_comment_based_help) that are generated into the scripts in the `..\exports` folder. Additionally, when writing custom cmdlets in the `..\custom` folder, you can use the help comments syntax, which decorate the exported scripts at build-time. The documentation examples are taken from the `..\examples` folder.
\ No newline at end of file
+The process of documentation generation loads `Az.PostgreSql` and analyzes the exported cmdlets from the module. It recognizes the [help comments](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help) that are generated into the scripts in the `..\exports` folder. Additionally, when writing custom cmdlets in the `..\custom` folder, you can use the help comments syntax, which decorate the exported scripts at build-time. The documentation examples are taken from the `..\examples` folder.
```

Comment:
@DaeunYim please revert the change to this readme.md

Reply:
sorry missed this,,!! fixed this